plugins/processors/reverse_dns/reversedns_test.go

```diff
@@ -54,3 +54,24 @@ func TestLoadingConfig(t *testing.T) {
 
 	require.Len(t, c.Processors, 1)
 }
+
```

Comment:
This test belongs in config/config_test.go instead of here.  TestConfig_LoadSpecialTypes is similar.

Reply:
you can't have config require reverse_dns due to cyclical dependencies. Since it mentions both, it has to go here.

Reply:
.. though, a config_test package might work.

Reply:
Looking at this with fresh eyes, the test above it is almost identical in setup. I'm not so sure that it belongs in config. The functionality in config is covered by another test (in config_test above), and this was just an extra sanity test I added to make sure the plugin had what it needed, and wasn't falling back on default values. In the end this was not needed (because it's not a regression test per se, and didn't highlight the bug I was fixing), but I figured it didn't hurt to add the test to the test suite. I'd recommend leaving it as is.

Reply:
I understand it's convenient to put it here because of dependencies, but it doesn't belong here because it's testing config parsing, not reverse dns functionality.  I'll leave it to you to move if you want.